src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Object/ObjectDefaultConverter.cs

```diff
@@ -268,6 +257,48 @@ internal override bool OnTryRead(ref Utf8JsonReader reader, Type typeToConvert,
             return true;
         }
 
+        internal static void PopulatePropertiesFastPath(object obj, JsonTypeInfo jsonTypeInfo, JsonSerializerOptions options, ref Utf8JsonReader reader, scoped ref ReadStack state)
```

Comment:
this code is just moved from the above because it needs to be reused elsewhere

Reply:
Where is it being reused? constructor converter presumably?